Makefile

```diff
@@ -23,10 +23,10 @@ endif
 
 grimd: 
 	go get ./...
-	go build $(LDFLAGS) -o $(GOPATH)/bin/grimd ./grimd
+	go install $(LDFLAGS) github.com/MediaMath/grim/grimd
 
 tmp/grimd-$(VERSION).zip: grimd | tmp 
-	zip -r -j $@ $(GOPATH)/bin/grimd
+	export PATH=$$PATH:$$GOPATH/bin; zip -r -j $@ $$(which grimd)
```

Comment:
However, that won't fix all GOPATH, I think we want this:

```
${GOPATH//://bin:}/bin
```

https://github.com/golang/go/wiki/GOPATH